src/_data/side-nav.yml

```diff
@@ -84,8 +84,6 @@
           permalink: /server/libraries
         - title: C & C++ interop
           permalink: /server/c-interop
-        - title: Native extensions
```

Comment:
I think we may want to keep (an updated version) of this page for the specialized use case of embedding Dart into existing native apps. Can we log a bug to remind ourselves to discuss that later?

Reply:
Done: #1548